src/RwaSpell.sol

```diff
@@ -0,0 +1,204 @@
+    address constant RWA_URN         = 0xC6172B516f265dF53123F052FAAEB2AD63e49df7;
+    address constant RWA_CONDUIT     = 0x0CF836924fD65Af0DE42294c8e9FAcCC19A384Dc;
+
+    // this is set to 10million right now
+    address constant PIP             = 0x0318D82C3b2a23d993dcE881aada122f311ca901;
+
+    // may be able to assign these as immutable
+    // try constants
+    DSValueAbstract constant pippip = DSValueAbstract(PIP);
+
+    RwaLiquidationLike constant oracle        = RwaLiquidationLike(PIP_RWA001);
+
+    uint256 constant SIX_PCT_RATE    = 1000000001847694957439350562;
+
+    // precision
+    uint256 constant public THOUSAND = 10 ** 3;
+    uint256 constant public MILLION  = 10 ** 6;
+    uint256 constant public WAD      = 10 ** 18;
+    uint256 constant public RAY      = 10 ** 27;
+    uint256 constant public RAD      = 10 ** 45;
+    
+    function execute() external {
+
+        ////////////////////////////////////////////////////////////////////////////////
+        // RWA-001 collateral deploy
+
+        // Set ilk bytes32 variable
+        bytes32 ilk = "RWA-001";
+
+        // add RWA-001 contract to the changelog
+        CHANGELOG.setAddress("RWA-001", RWA001_GEM);
+        CHANGELOG.setAddress("MCD_JOIN_RWA001", MCD_JOIN_RWA001);
+        CHANGELOG.setAddress("MCD_FLIP_RWA001", MCD_FLIP_RWA001);
+        CHANGELOG.setAddress("PIP_RWA001", PIP_RWA001);
+ 
+        // Sanity checks
+        require(GemJoinAbstract(MCD_JOIN_RWA001).vat() == MCD_VAT, "join-vat-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA001).ilk() == ilk, "join-ilk-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA001).gem() == RWA001_GEM, "join-gem-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA001).dec() == DSTokenAbstract(RWA001_GEM).decimals(), "join-dec-not-match");
+        require(FlipAbstract(MCD_FLIP_RWA001).vat()    == MCD_VAT, "flip-vat-not-match");
+        require(FlipAbstract(MCD_FLIP_RWA001).ilk()    == ilk, "flip-ilk-not-match");
+
+        // Set price feed for RWA001
+        SpotAbstract(MCD_SPOT).file(ilk, "pip", PIP);
+
+        // // // Set the RWA-001 flipper in the cat
+        CatAbstract(MCD_CAT).file(ilk, "flip", MCD_FLIP_RWA001);
+
+        // // // Init RWA-001 in Vat
+        VatAbstract(MCD_VAT).init(ilk);
+        // // // Init RWA-001 in Jug
+        JugAbstract(MCD_JUG).init(ilk);
+
+        // // // Allow RWA-001 Join to modify Vat registry
+        VatAbstract(MCD_VAT).rely(MCD_JOIN_RWA001);
+
+        // // // Allow RWA-001 Flipper on the Cat
+        CatAbstract(MCD_CAT).rely(MCD_FLIP_RWA001);
+
+        // // // Allow cat to kick auctions in RWA-001 Flipper
+        FlipAbstract(MCD_FLIP_RWA001).rely(MCD_CAT);
+
+        // // // Allow End to yank auctions in RWA-001 Flipper
+        // // TODO
+        FlipAbstract(MCD_FLIP_RWA001).rely(MCD_END);
+        
+        // since we're adding 2 collateral types in this spell, global line is at beginning
+        // TODO Line
```

Comment:
Missing global `Line`. Should be set as an increase now instead of absolute value due to IAM DC addition in previous spell.